sdk/core/azure-core-management/src/main/java/com/azure/core/management/PagedList.java

```diff
@@ -1,415 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
If this class is removed, what is the suggested replacement or usage for the paged list?

Reply:
Now the equivalent type is in azure-core - `PagedFlux` (async), `PagedIterable` (sync), hence this is no longer required in core mgmt.